python/runtime/db.py

```diff
@@ -401,3 +401,27 @@ def get_table_schema(conn, table):
             fields.append((field[0], field[1]))
         cursor.close()
     return fields
+
+
+def exec(conn, sql_stmt):
+    """Execute given sql statement and return if successful
```

Comment:
Execute given sql statement and return if successful -> Execute the given SQL statement and return true when success